### DISCUSSION TITLE
Chia auf eine andere Festplatte verschieben

### DISCUSSION THREAD

**Discussion by Kaecki** [Chia-Network/chia-blockchain#11382] [Support]
Hallo,
ich hatte Chia am Anfang ungewollt auf meiner Systemfestplatte mit 237 GB. Da Chia mittlerweile zu groß für die Festplatte ist, möchte ich dies gerne auf meine größere Festplatte "umziehen". Ich habe alle Dateien auf die andere Festplatte gezogen. Wenn ich Chia jetzt jedoch starte, synchronisiert er alles neu und Speichert es mir wieder auf meine Systemfestplatte. Wie bekomme ich es hin, dass es auf meiner größeren Festplatte gespeichert wird?

**Comment by Rapha0815** [comment]
Welches Betriebssystem?

**Comment by Kaecki** [comment]
Windows 10

> Am 01.05.2022 - 11:35 schrieb Rapha:
>
>
> Welches Betriebssystem?
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.
> Message ID: ***@***.***>

-------------------------------------------------------------------------------------------------
FreeMail powered by mail.de - MEHR SICHERHEIT, SERIOSITÄT UND KOMFORT

**Comment by loppefaaret** [comment]
https://spacefarmers.io/wiki/guides/farming/movedb
